spec/query_spec.cr

```diff
@@ -242,7 +242,7 @@ describe Avram::Query do
       user = UserBox.new.name("Mikias Abera").age(26).nickname("miki").create
       users = UserQuery.new.where("name = ? AND age = ?", "Mikias Abera", 26).where(:nickname, "miki")
 
-      users.query.statement.should eq "SELECT #{User::COLUMNS} FROM users WHERE nickname = $1 AND name = 'Mikias Abera' AND age = 26"
+      users.query.statement.should eq "SELECT #{User::COLUMNS} FROM users WHERE name = 'Mikias Abera' AND age = 26 AND nickname = $1"
```

Comment:
Now the clauses (when converted to SQL) should show up in the order that they're declared in the ORM syntax